### PR TITLE
Fixed map "Tread Lightly"

### LINKED ISSUE #9603: Map-placed actors in "Tread Lightly" have bogus health
They all define Health: 0 or 1. Looks like they might have been affected by a bad / multiply applied upgrade rule.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/27329366-map-placed-actors-in-tread-lightly-have-bogus-health?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Mailaender**
http://ra2maps.zzattack.org/Tiberian%20Sun/index.php?dir=Official%20Multiplayer%2FTread%20Lightly reimported with the fixed up version at https://github.com/OpenRA/OpenRA/pull/10368.

Closes https://github.com/OpenRA/OpenRA/issues/9603 and adds light spots as well as crates.

### CODE REVIEW

**Comment by pchote** [comment]
Map cordons look wrong here, too.  You can load the original map in FinalSun to see how the north and south edges should be adjusted.

**Comment by Mailaender** [comment]
Fixed the cordons (by fixing it in the importer) and removed the need for a hard-coded actor translation.

File: mods/ts/maps/tread_l/map.yaml [hunk 1]
```diff
@@ -78,1101 +78,1155 @@ Actors:
...
+	Actor342: crat01
+		Location: 123,20
+		Owner: Neutral
 
 Smudges:
 
 Rules:
 	World:
 		GlobalLightingPaletteEffect:
+			Red: 1
```
Comment (pchote) [inline mods/ts/maps/tread_l/map.yaml]: These should be omitted if they are 1.

Comment (Mailaender) [inline mods/ts/maps/tread_l/map.yaml]: Okay.

**Comment by pchote** [comment]
Cordons are still not correct :disappointed:

**Comment by pchote** [comment]
Looks good to me :+1:

**Comment by abcdefg30** [comment]
:+1: / :white_check_mark: here too.

**Pull request merged**